### PR TITLE
fix: await interactions rule scope.

### PR DESCRIPTION
**Pull request by zhyd1997**
Issue: #47 

## Checklist

Check the ones applicable to your change:

- [x] Ran `yarn update-all`
- [x] Tests are updated
- [x] Documentation is updated

## Change Type

Indicate the type of change your pull request is:

- [ ] `maintenance`
- [ ] `documentation`
- [x] `patch`
- [ ] `minor`
- [ ] `major`

### CODE REVIEW

File: tests/lib/rules/await-interactions.test.ts [hunk 1]
```diff
@@ -59,13 +59,12 @@ ruleTester.run('await-interactions', rule, {
...
-    //   import { userEvent } from '../utils'
-    //   Basic.play = async () => {
-    //     userEvent.click(canvas.getByRole('button'))
-    //   }
-    // `,
+    dedent`
+      import { userEvent } from '../utils'
+      Basic.play = async () => {
+        userEvent.click(canvas.getByRole('button'))
+      }
```
Comment (yannbf) [inline tests/lib/rules/await-interactions.test.ts:66]: Could you change this test case to the following. It's closer to the real use case

```suggestion
      import { userEvent } from '../utils'
      import { within } from '@storybook/testing-library'
      
      Basic.play = async (context) => {
        const canvas = within(context)
        userEvent.click(canvas.getByRole('button'))
      }
```

**Review by yannbf — CHANGES_REQUESTED** [review]
Hey @zhyd1997 thanks a lot for your contribution! I added a couple comments.

File: lib/configs/addon-interactions.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export = {
       },
     },
     {
-      files: ['main.@(js|cjs|mjs|ts)'],
+      files: ['.storybook/main.@(js|cjs|mjs|ts)'],
```
Comment (ndelangen) [inline lib/configs/addon-interactions.ts:20]: These changes don't quite seem related to the PR, @zhyd1997 what prompted you to make these 3 changes?

Comment (yannbf) [reply lib/configs/addon-interactions.ts:20]: This is very likely correct. It's possible a previous PR didn't update these, and while they ran `yarn update-all` this got updated

**Comment by zhyd1997** [comment]
@yannbf @ndelangen 

Thanks for the quick code review, give me some more time to update the PR.

File: lib/rules/await-interactions.ts [hunk 3]
```diff
@@ -130,24 +130,36 @@ export = createStorybookRule({
       return getClosestFunctionAncestor(parent)
     }
 
+    const isExpectFromStorybookImported = (node: ImportDeclaration) => {
+      return (
+        node.source.value.startsWith('@storybook/')
```
[the remark this answers is not in the document]
Comment (zhyd1997) [reply lib/rules/await-interactions.ts]: still working on **case 2**...

[the remark this answers is not in the document]
Comment (zhyd1997) [reply lib/rules/await-interactions.ts]: Done https://github.com/storybookjs/eslint-plugin-storybook/pull/100/commits/c5ed780196496394bfdeacc9e8b3ae0652b2368a

**Comment by ndelangen** [comment]
Is this ready for review again?

If so, @yannbf and me can take a look again? 🙏 
Thank you for the hard work @zhyd1997 !

**Comment by zhyd1997** [comment]
@ndelangen 

Yes, it's ready for review again.
Sorry for that! 🙏🏿

**Review by yannbf — APPROVED** [review]

**Comment by yannbf** [comment]
Thank you so much once again @zhyd1997 ! Sorry it took some time, but I'm getting this in now!

**Pull request merged**